models/DataObject/ClassDefinition/Data/QuantityValue.php

```diff
@@ -430,15 +434,26 @@ public function getForCsvExport($object, $params = [])
      */
     public function getFromCsvImport($importValue, $object = null, $params = [])
     {
-        $values = explode('_', $importValue);
+        if (strpos($importValue, '_') !== false) {
+            [$number, $unitId] = explode('_', $importValue);
+            $number = (float) str_replace(',', '.', $number);
 
-        $value = null;
-        if ($values[0] && $values[1]) {
-            $number = (float) str_replace(',', '.', $values[0]);
-            $value = new Model\DataObject\Data\QuantityValue($number, $values[1]);
+            return new Model\DataObject\Data\QuantityValue($number, $unitId);
         }
 
-        return $value;
+        if (strpos($importValue, ' ') !== false) {
+            [$number, $abbreviation] = explode(' ', $importValue);
+            $number = (float)str_replace(',', '.', $number);
+            $unit = Model\DataObject\QuantityValue\Unit::getByAbbreviation($abbreviation);
```

Comment:
I think for compatibility reasons we should still support the import with unit IDs, shouldn't be an issue, right?

Reply:
Is there in lines 437-442

Reply:
🤦‍♂️ Sorry - thanks a lot! 😊